addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/ssti/SstiUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.ascanrulesAlpha.ssti;
+
+import java.security.SecureRandom;
+
+/** @author DiogoMRSilva (2018) */
+public class SstiUtils {
+
+    private SstiUtils() {
+        // Utility class
+    }
+
+    /** from XXE scna rule (yhawke) */
+    public static String randomStringFromAlphabet(String alphabet, int length) {
```

Comment:
Maybe this should be moved to `commonlib` if it's used in multiple places.

Reply:
I'll tackle this in another PR.

Reply:
It could also use `RandomStringUtils`, which seems to be the one more used throughout.

Reply:
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/RandomStringUtils.html#randomAlphanumeric-int-